book/src/developers/designs/application_passwords.md

```diff
@@ -161,55 +154,61 @@ needed.
 
 Since application passwords are related to applications, on delete of an
 application all entries that have a bound application password should be removed
-from user accounts.
+from user accounts. The associated group will be automatically deleted too.
 
 ### Access controls
 
-The IDM administrators group can manage the applications and user's application
-passwords.
+The "Application administrators" group will manage the applications, and
+applications will allow "managed by" so that they can have delegated
+administration.
 
-Users can manage their own application passwords.
+The "Application user passwords administrators" group will be able to list the
+users's application passwords and delete them but only the users will be able
```

Comment:
If we add the enable/disable flag, they should be able to enable or disable them too.